scripts/download_openmeteo_2024_2025.py

```
@@ -0,0 +1,151 @@
+                    if interval > 0:
+                        time.sleep(min(300, remaining_wait - (interval * 300)))
+                        time_left = remaining_wait - (interval * 300) - 300
+                        if time_left > 0:
+                            print(f"     {int(time_left/60)}m {int(time_left%60)}s remaining...", end='\r')
+                
+                # Final short sleep for any remaining seconds
+                final_wait = remaining_wait % 300
+                if final_wait > 0:
+                    time.sleep(final_wait)
+            else:
+                print(f"  Download took longer than wait period, proceeding immediately")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)